src/components/ReanimatedSwipeable.tsx

```diff
@@ -446,7 +456,6 @@ const Swipeable = forwardRef<SwipeableMethods, SwipeableProps>(
       },
       openRight() {
         'worklet';
-        rightWidth.value = rowWidth.value - rightOffset.value;
```

Comment:
Shouldn't we change it to `updateRightElementWidth`?

Reply:
There's no need, this line is now present in `calculateCurrentOffset()` which is executed in the next line before anything else.